README.md

```diff
@@ -20,52 +20,55 @@ The client jar is distributed via maven central, and can be downloaded [here](ht
 </dependency>
 ```
 
-## Usage
+### Unix Domain Socket support
+
+As an alternative to UDP, Agent6 can receive metrics via a UNIX Socket (on Linux only). This library supports transmission via this protocol. To use it, simply pass the socket path as a hostname, and 0 as port.
+
+By default, all exceptions are ignored, mimicking UDP behaviour. When using Unix Sockets, transmission errors trigger exceptions you can choose to handle by passing a `StatsDClientErrorHandler`:
+
+- Connection error because of an invalid/missing socket will trigger a `java.io.IOException: No such file or directory`
+- If dogstatsd's reception buffer were to fill up, the send will timeout after 100ms and throw either a
```

Comment:
Just to clarify: this is `true` if the user is using a non blocking client and is using the default timeout of 100ms. Howeever, a power user could use a non-blocking client and set a different timeout, or a timeout of zero in which case the client would block if the buffer were full. With a blocking client, the client will block for obvious reasons.

Reply:
I updated the sentence a bit, but i think we can skip the power user use-case for now. If we see support tickets related to it we will update this doc accordingly.